sdk/datafactory/Azure.ResourceManager.DataFactory/tests/DataFactoryManagementTestBase.cs

```diff
@@ -45,7 +46,7 @@ protected async Task<ResourceGroupResource> CreateResourceGroup(SubscriptionReso
 
         protected async Task<DataFactoryResource> CreateDataFactory(ResourceGroupResource resourceGroup, string dataFactoryName)
         {
-            DataFactoryData data = new DataFactoryData(resourceGroup.Data.Location);
+            DataFactoryData data = new DataFactoryData(AzureLocation.WestUS2);
```

Comment:
why not use env variable?

Reply:
That's because I personally prefer to set the value directly.